Purchases/SwiftSources/LocalReceiptParsing/LocalReceiptParser.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  LocalReceiptParser.swift
+//  Purchases
+//
+//  Created by Andrés Boedo on 6/29/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+@objc public enum LocalReceiptParserErrorCode: Int {
+    case ReceiptNotFound,
+         UnknownError
+}
+
+private enum IntroEligibilityStatus: Int {
+    case unknown,
+         eligible,
+         ineligible
+}
```

Comment:
ended up going with this, I spent the day trying out like 4285 approaches and this was the winner.